.env

```diff
@@ -44,6 +44,7 @@ ANALYTICS_URL=http://localhost:5000
 NJORD_ORIGIN=http://njord-transactions-server
 FREYJA_ORIGIN=http://localhost:7800
 SKADI_ORIGIN=http://localhost:8080
+SKADI_API_ORIGIN=http://skadi-api-server.local.svc.cluster.local
```

Comment:
Probably didn't mean to expose like this right?

Reply:
this is local URL, its fine, we can obfuscate the ones in prod file, but also not mandatory

Reply:
It's fine, it's just the full dns name inside cluster, short hand would just be `skadi-api-server`.